obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -1,10 +1,36 @@
 ---
+- block:
+    - name: 'Set dist from commandline args'
+      set_fact:
+        build_srpm_tito_releaser_args: "{{ build_srpm_tito_releaser_args | default([]) }} + [ '--dist {{ build_srpm_dist }}' ]"
+      when: build_srpm_dist is defined
+
+    - name: 'Set scl from commandline args'
+      set_fact:
+        build_srpm_tito_releaser_args: "{{ build_srpm_tito_releaser_args | default([]) }} + [ '--scl {{ build_srpm_scl }}' ]"
```

Comment:
given `build_srpm_tito_releaser_args` is defined in `defaults`, I think we don't need the `| default([])` here?